Platform package

danielmarbach opened the pull request:
This shows how the Platform package simplifies the monitoring demo and allows to buid the monitoring demo with CI/CD and release the artifacts only as a downloadable zip package

Before Merge:

* [x] 1.0.0 (non-beta) package of Particular.PlatformSample
* [x] Release pipeline to put binary output on S3
* [x] Update monitoring demo to link to S3 zip file https://github.com/Particular/docs.particular.net/pull/4184https://github.com/Particular/docs.particular.net/pull/4184

DavidBoike commented:
@Particular/monitoring-maintainers 	We are ready to deploy this, but would like your sign-off first.

Please don't merge it, as that will make the zipball the download currently points to unusable. We want to update the tutorial to point to the version hosted on S3 as a result of the [build deploy pipeline](https://builds.particular.net/project.html?projectId=MonitoringDemo&tab=projectOverview) we built for it.

Just review it and give your :+1: - we'll handle the rest.

mikeminutillo commented:
I might be doing something wrong but I can't figure out how to get this running. When I run the powershell in the root (which does not have a corresponding batch file so I need to command line to it) I get all sorts of errors related to working directory:

```
PS C:\code\Particular\monitoringdemo> .\Run.ps1
Starting demo
Starting the Particular Platform
Start-Process : This command cannot be run because either the parameter "WorkingDirectory" has a value that is not
valid or cannot be used with this command. Give a valid input and Run your command again.
At C:\code\Particular\monitoringdemo\Run.ps1:28 char:15
+ ... $platform = Start-Process ".\Platform\net461\Platform.exe" -WorkingDi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartProcessCommand

Once ServiceControl has finished starting a browser window will pop up showing the ServicePulse monitoring tab
Starting Demo Solution
Start-Process : This command cannot be run because either the parameter "WorkingDirectory" has a value that is not
valid or cannot be used with this command. Give a valid input and Run your command again.
At C:\code\Particular\monitoringdemo\Run.ps1:33 char:14
+ ...  $billing = Start-Process ".\Billing\net461\Billing.exe" -WorkingDire ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartProcessCommand

Start-Process : This command cannot be run because either the parameter "WorkingDirectory" has a value that is not
valid or cannot be used with this command. Give a valid input and Run your command again.
At C:\code\Particular\monitoringdemo\Run.ps1:34 char:12
+   $sales = Start-Process ".\Sales\net461\Sales.exe" -WorkingDirectory ...
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartProcessCommand

Start-Process : This command cannot be run because either the parameter "WorkingDirectory" has a value that is not
valid or cannot be used with this command. Give a valid input and Run your command again.
At C:\code\Particular\monitoringdemo\Run.ps1:35 char:15
+ ... $shipping = Start-Process ".\Shipping\net461\Shipping.exe" -WorkingDi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartProcessCommand

Start-Process : This command cannot be run because either the parameter "WorkingDirectory" has a value that is not
valid or cannot be used with this command. Give a valid input and Run your command again.
At C:\code\Particular\monitoringdemo\Run.ps1:36 char:15
+ ... $clientUI = Start-Process ".\ClientUI\net461\ClientUI.exe" -WorkingDi ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartProcessCommand

Press ENTER to shutdown demo
```

When I try to run the sln I get an Access Denied error starting ServiceControl.

What am I doing wrong?

danielmarbach commented:
Talked to @mikeminutillo about the problem above. There were two issues. One issue was that the readme was not up2date and he was using the wrong powershell script to start things and he hadn't compiled the solution. I updated the readme. Another problem was related to him having ExposeRavenDB in the registry which gets picked up automatically by ServiceControl. I fixed that by releasing 1.0.1 of the platform sample that hardcodes the value to false
